src/config/config_set.rs

```diff
@@ -11,6 +12,34 @@ pub struct ConfigSet {
     pub extensions: Extensions,
 }
 
+#[derive(Clone, Debug, Default)]
+pub struct FileDescriptorSetWithId {
+    pub id: Option<String>,
+    pub file_descriptor_set: FileDescriptorSet,
+}
+
+#[derive(Clone, Debug)]
+pub struct ServiceDocumentWithId {
+    pub id: Option<String>,
+    pub service_document: ServiceDocument,
+}
+
+// Is necessary to implement Default for ServiceDocumentWithId because ServiceDocument does not implement Default
+impl Default for ServiceDocumentWithId {
+    fn default() -> Self {
+        ServiceDocumentWithId {
+            id: None,
+            service_document: ServiceDocument { definitions: vec![] },
+        }
+    }
+}
+
+#[derive(Clone, Debug, Default)]
+pub struct RawContent {
+    pub id: Option<String>,
+    pub content: async_graphql_value::ConstValue,
+}
```

Comment:
I think we can create a polymorphic container 

```rust
pub struct Asset<A> {
    id: Option<string>,
    content: A
}
```

Reply:
we don't need it anymore as we won't support `Data` (this was only used by `Data` type

Reply:
now I got it

Reply:
I've used the `Content` name, but let me know if you prefer `Asset`; also, great suggestion, thank you :smile: